components/zcash_address/src/kind/unified/address.rs

```diff
@@ -164,8 +164,9 @@ mod tests {
     }
 
     fn arb_unified_address_for_typecodes(
-        typecodes: Vec<Typecode>,
+        mut typecodes: Vec<Typecode>,
     ) -> impl Strategy<Value = Vec<Receiver>> {
+        typecodes.sort_unstable_by_key(|tc| <u32>::from(*tc));
```

Comment:
Should this be cloning `typecodes` before mutating it? It doesn't seem common for `arb_*` functions to mutate their arguments, and I'm not sure whether there are any special requirements to make shrinking work.

Reply:
Since this takes ownership, there's no need to clone it, right?